deploy/backplane/cee/01-cee-machineset-ClusterRole.yml

```diff
@@ -0,0 +1,23 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: backplane-cee-machineset-readers
```

Comment:
s/machineset/machine ?
Cause it's not only machineset, it's all machine related.

Reply:
also please rename the file name.